flink-connectors/flink-connector-rabbitmq2/README.md

```diff
@@ -19,6 +19,143 @@ and the Sink API specified in [FLIP-143](https://cwiki.apache.org/confluence/dis
 
 For more information about RabbitMQ visit https://www.rabbitmq.com/.
 
-In order to view how to use the connector inspect
-[Source](src/main/java/org/apache/flink/connector/rabbitmq2/source/README.md) and 
-[Sink](src/main/java/org/apache/flink/connector/rabbitmq2/sink/README.md).
+# RabbitMQ Source
+
+Flink's RabbitMQ connector provides a streaming-only source which enables you to receive messages
```

Comment:
Why RabbitMQ connector provide a streaming only source? Doesn't support the batch mode?